rplugin/python3/floobits/__init__.py

```diff
@@ -9,12 +9,12 @@
 import neovim
 
 try:
-    unicode()
+    str()
 except NameError:
-    unicode = str
+    str = str
```

Comment:
Looks like this can be dropped. I am guessing it is only there because of `s/unicode/str/g`

Reply:
yep, I think it's an artifact of 2to3